### ISSUE TITLE
Training from Scratch

### ISSUE THREAD

**Issue by Mahsa13473** [laughtervv/DISN#6]
Hi there,

Thanks for releasing the codes, it is amazing work!
I try to train the network from scratch and follow all the steps that were mentioned in Readme file, but I couldn't get the same results in comparison to pretrained model.

I was wondering which hyperparameters are used for the pretarined one. Is it the same as the defaults in train_sdf.py?
How many epochs did you train to get the best accuracy?
Also which dataset was used for training? The old one or the new one that you mentioned in Readme?

**Comment by no-materials** [comment]
Hello, in addition to @Mahsa13473 's comment, can you also provide the approximate training time?
